### PR TITLE
Feature/news stories initial sass layout

### LINKED ISSUE #31: Create CSS/Sass in NewStories Component

### PR DESCRIPTION
**Pull request by jordan-sullivan**
Hi Team, Please review this PR. 

## Description of Changes:
*Added basic Sass to news stories page. 
*Sized stories for responsive design to be able to scroll coherently with background static. 
*Added conditional for default image 

## Put an 'X' next to all that apply
New Test: [x]
New Feature:[x]
Bug Fix: []

## Before You Submit Ensure
All Tests Pass: [x]
Full Coverage: [x]
Runs Locally: [x]

Closes #31 #34 

Thanks!

### CODE REVIEW

File: src/components/NewsStories/NewsStories.js [hunk 1]
```diff
@@ -2,34 +2,46 @@ import React from "react";
...
-          <div className="action-box">
-            <p className="action">TAKE ACTION</p>
-          </div>
-          <div className="title-box">
-          <h2 className="news-story-title">{story.title}</h2>
-            <p className="author-name">by {story.author}</p>
-          </div>
-        </section>
+      <div className="story">
+        {!story.img ?
```
Comment (camianderson) [inline src/components/NewsStories/NewsStories.js:17]: Nice work with the conditional rendering.

File: src/components/NewsStories/NewsStories.scss [hunk 2]
```diff
@@ -1,24 +1,75 @@
...
+    justify-content: center;
+    margin-bottom: 1rem;
+    margin-top: 2rem;
+    width: 100vw;
+    -webkit-overflow-scrolling: touch;
 
-.news-story-title {
-    border: 1px sold black;
-}
+    &__section{
```
Comment (camianderson) [inline src/components/NewsStories/NewsStories.scss:12]: Good job changing the classes names to comply with the BEM convention.

**Comment by camianderson** [comment]
Good job! Next step is to work on filtering the data according to the selected topic.

**Review by Neil-B-Linton — APPROVED** [review]
Looks great glad to see this-looks amazing!

**Pull request merged**